### ISSUE TITLE
Header Information is not populated in the text file

### ISSUE THREAD

**Issue by bigboss97** [SarutaSan72/Yass#65]
I'm quite new in Yass (v2.1.1 Win10) and I'm trying to create a Chinese song.
These are the information I put in and each step went through without error:
![Screenshot - 20-05-06 16 46 16](https://user-images.githubusercontent.com/34834507/81145890-94b80b80-8fba-11ea-88bb-dd34dfeaec06.png)
Directory and files with unicode are generated correctly. Only the header doesn't contain the information I put in:
![Screenshot - 20-05-06 16 49 28](https://user-images.githubusercontent.com/34834507/81146005-d21c9900-8fba-11ea-8022-8fbb383888aa.png)
After I fixed the text file manually. It works.

**Comment by SarutaSan72** [comment]
I just tested it with the attached file and it works for me. Please compare your file encoding with mine, or simply use my file as base for your song. 
[Test - Chinese.txt](https://github.com/SarutaSan72/Yass/files/4587430/Test.-.Chinese.txt)

**Comment by bigboss97** [comment]
Yes, Yass has no problem in loading Chinese. As I mentioned, after I manually fixing the text file. There's no problems.
The problem occurs when I select "New" and enter the title and artist information via dialog box, see previous screenshot. Everything was generated correctly, even the Chinese lyrics. Only the header information wasn't populated.
Step to re-produce:
1) Select "New...", select MIDI etc.
2) Select a MP3, e.g. "姚蘇蓉 - 我與咖啡.mp3"
3) In Header Information dialog, the Title and Artist fields are populated correctly. Enter Genre, Language and BPM. Continue to the end.

Result: Directory and files are generated correctly. But the header information is not written in the text file, see screenshot in previous post.

**Comment by SarutaSan72** [comment]
For now I cannot reproduce this. I tried "New..." without MIDI, without MP3, with and without UTF-8, and entered TITLE/ARTIST manually.  All fine.

Could you please mail me your MP3 file? Maybe the MP3 metadata extraction fails. 

-->  mail@yass-along.com

**Comment by bigboss97** [comment]
That's strange. I'm using v2.1.1.
These are my settings:
![Screenshot - 20-05-08 08 56 36](https://user-images.githubusercontent.com/34834507/81352661-72350800-910a-11ea-87d4-a5c96cd79c4c.png)
I'm pretty sure I've also seen the header populated before. When I started I found v2.0.1 on net. I tried without lyrics first and found question marks in the header. Since I know Vocaluxe requires BOM UTF-8 I changed the settings to above.
Now I'm not sure whether I saw the header in v2.0.1.
